packages/runtime/src/widgets/Search.tsx

```diff
@@ -28,6 +28,7 @@ const widgetName = "Search";
 export type SearchProps = {
   placeholder?: string;
   searchKey?: string;
+  searchWidget?: { [key: string]: unknown };
```

Comment:
I'm not sure if `searchWidget` is the right naming for this property...this should be like resultWidget maybe?